src/main/java/duke/exception/WrongCommandFormatException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exception;
+
+public class WrongCommandFormatException extends Exception{
```

Comment:
Perhaps there can be a header comment here to inform users on what the class does. Similar comment for the other classes, if any, lacking a comment header

Reply:
Oops missed that out. Thanks for spotting it.